src/controllers/livetvstatus.js

```diff
@@ -168,9 +168,6 @@ define(["jQuery", "globalize", "scripts/taskbutton", "dom", "libraryMenu", "layo
         }), menuItems.push({
             name: "Xml TV",
             id: "xmltv"
-        }), menuItems.push({
-            name: globalize.translate("ButtonOther"),
-            id: "other"
```

Comment:
The corresponding action for this key needs to be removed as well, along with any unused translations.